### ISSUE TITLE
It seems to forget any favourite and previous connections at every restart

### ISSUE THREAD

**Issue by IGJAmpere** [flathub/com.mongodb.Compass#9]
Using version 1.33.1. 

You do not even need to restart, just disconnect from your current connection.

**Comment by mijorus** [comment]
Could you attach the log file it generates? You can find it from `Help > Open Log file`.

I am daily driving it and I use favorites without issues

**Comment by IGJAmpere** [comment]
[delete.txt](https://github.com/flathub/com.mongodb.Compass/files/9757719/delete.txt)

**Comment by mijorus** [comment]
I used to have this problem and it was related to `libsecret`, but it was supposed to be fixed. 

This needs investigation.

@IGJAmpere  Could you tell me some details about your OS, maybe I can try to replicate the issue...

**Comment by IGJAmpere** [comment]
Yes, I am using a fully updated Debian 11, with backports, including Flatpak 1.14.0.

I'll try to install it on my Arch computer, too. 

It is no big deal for me, as of right now I simply launch the default connection and I can work fine. But hey, we must report the bugs.

**Comment by mijorus** [comment]
The issue you are reporting is nothing new, take a look at the following link:
https://github.com/flathub/flathub/pull/3269#issuecomment-1156400597

**Comment by dominikzogg** [comment]
Hello @IGJAmpere is this still an issue?

**Comment by IGJAmpere** [comment]
Hi! 
I just checked, and it seems to work now, :heavy_check_mark: in fact, it is showing now all the connections I tried to save (and it seems that I actually did) during those months.

My only issue now (which might not be related to the Flatpak packaging) is that these favourited connections only show up when I relaunch the app. 

Here's my info, if it helps anyone: 
- Operating System: Debian GNU/Linux 12
- Kernel Version: 6.1.0-9-amd64 (64-bit)
- Flatpak version: v1.14.4
- Compass version: v1.38.0

Thank you!
